code-studio/src/js/components/progress/progress_dot.jsx

```diff
@@ -127,10 +129,10 @@ const styles = {
  */
 const ProgressDot = React.createClass({
   propTypes: {
-    levels: levelProgressShape,
+    level: levelProgressShape.isRequired,
```

Comment:
Looks like `s/levels/level` is just a typo fix, was used as `level` everywhere already?